modules/ROOT/pages/getting-started.adoc

```diff
@@ -1,142 +1,67 @@
 :experimental:
 = Fedora CoreOS - Getting Started
 
-Fedora CoreOS has no install-time configuration. Every Fedora CoreOS system begins with a generic, unconfigured disk image. On first boot Ignition will read the supplied config and configure the system. Ignition configs are usually supplied via the cloud’s userdata mechanism, or, in the case of bare metal, injected at install time. This guide will show you how to launch Fedora CoreOS on AWS, QEMU, and bare metal as well as how to create Ignition configs.
+== Background
+
+Fedora CoreOS (FCOS) has no install-time configuration. Every FCOS system begins with a generic disk image. For each deployment mechanism (bare metal, local VM, or cloud VM), the configuration must be supplied at first boot. This is done via a configuration file and FCOS Ignition. For cloud deployments, Ignition supplies the configuration via the cloud’s userdata mechanism. In the case of bare metal, Ignition injects the configuration at install time.
+
+This guide will show you how to launch Fedora CoreOS on AWS, QEMU, and bare metal.
+
+=== Prerequisite
+Before you can launch FCOS, you must have an Ignition configuration file. To produce an Ignition file, see xref:producing-ign.adoc[Producing an Ignition File].
```

Comment:
Not sure if we have a style guide, but I've been trained to avoid direct "you". I usually stick to https://github.com/coreos/docs/blob/master/STYLE.md as a general guide.

Reply:
In academic writing, it is best to avoid "you." However, for clarity in technical documentation, I prefer to use you to avoid awkwardness, circumlocution, and worst of all, the passive voice. I can, however, rewrite this section to be more elegant than it is.

Reply:
Ack, feel free to keep your preferred wording then.